client/src/components/CreateMeetup/CreateMeetup.js

```diff
@@ -1,7 +1,246 @@
-import React from "react";
+import { React, useState } from "react";
+import makeStyles from "@material-ui/styles/makeStyles";
+import {
+  Box,
+  Button,
+  Container,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  Typography,
+} from "@material-ui/core";
+import AutoComplete from "@material-ui/lab/Autocomplete";
+import TextField from "@material-ui/core/TextField";
+import {
+  KeyboardDatePicker,
+  KeyboardTimePicker,
+  MuiPickersUtilsProvider,
+} from "@material-ui/pickers";
+import DayUtils from "@date-io/dayjs";
+
+const useStyles = makeStyles(() => ({
+  formSection: {
+    marginTop: 15,
+    marginBottom: 15,
+  },
+  formInput: {
+    width: 200,
+  },
+}));
 
 function CreateMeetup() {
-  return <>Create meetup</>;
+  const classes = useStyles();
+  const [name, setName] = useState("");
+  const [date, setDate] = useState(Date.now());
+  const [meetupType, setMeetupType] = useState("");
+  const [meetupLocation, setMeetupLocation] = useState("");
+  const [groupMin, setGroupMin] = useState("");
+  const [groupMax, setGroupMax] = useState("");
```

Comment:
Why are these strings?

Reply:
Good question :^)

I'll change the `groupMin` and `groupMax` to be numbers. What would you suggest for the `meetupType` and `meetupLocation`? I was thinking `meetupLocation` would be a string because it would represent a remote meetup location and could be a Zoom link or something, and we could have a separate state for the in-person meetups. Thoughts?